NuKeeper.Tests/Engine/RepositoryUpdaterTests.cs

```diff
@@ -28,12 +33,11 @@ public class RepositoryUpdaterTests
         [Test]
         public async Task WhenThereAreNoUpdates_CountIsZero()
         {
-            var packageUpdater = Substitute.For<IPackageUpdater>();
             var updateSelection = Substitute.For<IPackageUpdateSelection>();
             UpdateSelectionAll(updateSelection);
 
-            var repoUpdater = MakeRepositoryUpdater(
-                packageUpdater, updateSelection,
+            var (repoUpdater, packageUpdater) = MakeRepositoryUpdater(
```

Comment:
New syntax?

Reply:
Tuple deconstruction. Added in 7.Something